developer-handbook/github-process/post-code-review.md

```diff
@@ -0,0 +1,19 @@
+# Post Code Review
+
+Your Pull Request was reviewed by a colleague (or a few colleagues, which is great). What's left to do now is:
+
+## Address it
+- Just like in the code review step, we opt for friendly tones in their replies as well! We all have the best intentions for our codebases and every input is valuable.
```

Comment:
should this be 'in _the_ replies as well'?

Reply:
I meant it like "the code reviews' replies", but am open to correcting as I don't know which one is correct? 😅

Reply:
Ah, that makes sense too. I just didn't read it that way. Happy to keep it as is 😄

Reply:
OK! Resolving